recipes/crispector/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "1.0.2b6" %}
+
+package:
+  name: crispector
+  version: "{{ version }}"
+
+build:
+  number: 0
+  entry_points:
+    - crispector = crispector.cli:main
+  noarch: generic
```

Comment:
Sorry I haven’t seen this in the previous round: This should be `noarch: python`.

Reply:
Done